src/util/widgethelper.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <QScreen>
+
+namespace mixxx {
+namespace WidgetHelper {
```

Comment:
Namespace names should be lowercase by convention. Alternatively we could use a utility class with a static method. Then you don't need to add the `inline` keyword.

Reply:
A class that only has one static method doesn't make much sense.